webpack.config.js

```diff
@@ -36,16 +39,21 @@ function getLoadersConfig(env) {
 }
 
 function getPluginsConfig(env) {
+  if(!config.availableLanguages) {
+    throw "availableLanguages needs to be configured in the config file";
+  }
+  var languageExpression = new RegExp("^./(" + config.availableLanguages.join('|') + ")$");
```

Comment:
This requires that we specify a config file during build-time, which is something we don't want to do. Maybe we could loop through all config files and gather the files from there.